src/runonce-generator

```diff
@@ -29,20 +29,35 @@
 RUNONCE_TARGETS="/usr/share/runonce/targets"
 RUNONCE_QUEUED_STAMP_DIRECTORY="/var/lib/runonce/queue"
 RUNONCE_QUEUED_STAMP="/var/lib/runonce/runonce_queued" # Only used by the systemd-generator
+RUNONCE_QUEUED_USER="droidian"
```

Comment:
can we determine that automatically? Perhaps by looking at the groups of the user (like, if the user has UID >= 1000 and a shell configured, it's likely the target user).

This I think might also help us in future if we want to enable multiple users

Reply:
Yes, should be easy to add, will look at it tomorrow.

Reply:
Code updated to handle multiple users.